### PR TITLE
feat(onboarding): add connection type descriptions

### LINKED ISSUE #676: Make it more clear to users which network they are connecting/connected to (testnet vs mainnet)
Installing Zap on Win10 Pro. Right after the install, created the wallet and suggested to fund it. Bitcoin address (both QR and text) are invalid bitcoin address. This: 2MtYDfkWfWvzia1oA5M8mdxprHyRi97bka1

### PR DESCRIPTION
**Pull request by mrfelton**
## Description:

Add descriptions to each connection type to help users understand what each of the different options do.

Fix #676

## Motivation and Context:

Help users to understand what the different connection types do

## How Has This Been Tested?

Open up the app and review the first onboarding screen.

## Types of changes:

New feature

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have reviewed and updated the documentation accordingly.
- [x] I have read the _CONTRIBUTING_ document.
- [ ] I have added tests to cover my changes where needed.
- [ ] All new and existing tests passed.
- [x] My commits have been squashed into a concise set of changes.

### CODE REVIEW

**Comment by mrfelton** [comment]
@JimmyMow I've tried 2 different things here to address the concerns in #676

1. Added new "testnet" superscript to the `default` button
2. Added additional help text for each option.

Whilst I think the "testnet" superscript is possibly enough to help users understand that the default mode is testnet only, I felt that it would be even more useful if all of the options had some additional help text and not only the default option.

With the additional help text in place, potentially the "testnet" superscript could actually be removed.

File: app/components/Onboarding/ConnectionType.js [hunk 1]
```diff
@@ -21,12 +28,19 @@ const ConnectionType = ({ connectionType, setConnectionType }) => (
...
+        for advanced users only.
+      </div>
     </section>
     <section className={`${styles.option} ${connectionType === 'btcpayserver' && styles.active}`}>
       <div className={`${styles.button}`} onClick={() => setConnectionType('btcpayserver')}>
         {connectionType === 'btcpayserver' ? <FaCircle /> : <FaCircleThin />}
         <span className={styles.label}>BTCPay Server</span>
       </div>
+      <div className={`${styles.description}`}>
+        Connect to your own BTCPay Server instance in order access you BTCPay Server wallet.
```
Comment (JimmyMow) [inline app/components/Onboarding/ConnectionType.js]: in order access `your`** not you

**Comment by JimmyMow** [comment]
@mrfelton one quick typo fix before we can merge this. Concept ACK

File: app/components/Onboarding/ConnectionType.js (hunk 1 above)
Comment (mrfelton) [reply app/components/Onboarding/ConnectionType.js]: good catch. Fixed

**Comment by JimmyMow** [comment]
Tested ACK 746038c

**Pull request merged**